pkgs/servers/roundcube/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```

Comment:
this ```stdenv.mkDerivation``` can be changed to ```fetchurl``` or ```fetchzip``` (with ```installPhase``` renamed to ```postFetch```)

Then, besides decreasing the number of code lines, there will be a single ```roundcube``` fixed-output derivation for all architectures and platforms.

Reply:
OK. I will try to change that